root/usr/bin/get-zip-server_config

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+# Script to extract the NEXT_SERVER value from the incoming DHCP lease
+# and write it to a place where systemd/profile.d can source it
+
+# NetworkManager automatically strips ".sh" and appends "_config" on the source
+# file. This is only run when the interface is going "up" so we don't interfere
+# with the next-server settings on "down"
```

Comment:
Interesting, why you didn't put this into the `.sh` script directly? What is the advantage here?